stake-pool/program/src/instruction.rs

```diff
@@ -1323,6 +1323,64 @@ pub fn decrease_validator_stake_with_vote(
     )
 }
 
+/// Create a `IncreaseAdditionalValidatorStake` instruction given an existing
+/// stake pool, valiator list and vote account
+pub fn increase_additional_validator_stake_with_list(
+    program_id: &Pubkey,
+    stake_pool: &StakePool,
+    validator_list: &ValidatorList,
+    stake_pool_address: &Pubkey,
+    vote_account_address: &Pubkey,
+    lamports: u64,
+    ephemeral_stake_seed: u64,
+) -> Instruction {
+    let validators = validator_list
```

Comment:
nit, since it's only finding one, let's call this `validator` or `validator_info`
```suggestion
    let validator_info = validator_list
```